.github/workflows/package.yml

```diff
@@ -67,6 +67,7 @@ jobs:
 
       - name: Package (windows)
         if: ${{ runner.os == 'Windows' }}
+        shell: bash
```

Comment:
I am not 1000% sure that this will work as expected, but I feel like it's better to make things run in bash than to add the windows witchcraft where it's not necessary.

I guess maybe it would be good to have a workflow that audits our `yml` files for multi-line things that would run on windows. but that seems hard to do.